packages/components-storybook/src/stories/components/buttons/inverse.tsx

```diff
@@ -0,0 +1,290 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, H1, H5 } from '@trezor/components';
+import { storiesOf } from '@storybook/react';
+
+const Wrapper = styled.div`
+    padding: 1.6rem;
+`;
+
+const Row = styled.div`
+    display: flex;
+    margin: 0.5rem 0 2rem;
+    flex-wrap: wrap;
+
+    button {
+        margin: 10px 10px;
+    }
+`;
+
+const Div = styled.div`
+    margin: 0.5rem 0 2rem;
+`;
+const SnapshotWrapper = styled.div`
+    display: inline-flex;
+`;
+const SnapshotWrapperDiv = styled.div``;
+
+storiesOf('Buttons', module).add(
+    'Inverse',
+    () => (
+        <Wrapper>
+            <H1>Inverse </H1>
+            <Row>
+                <SnapshotWrapper data-test="button_inverse">
+                    <Button isInverse variant="success">
+                        Button
+                    </Button>
+                    <Button isInverse variant="info">
+                        Button
+                    </Button>
+                    <Button isInverse variant="warning">
+                        Button
+                    </Button>
+                    <Button isInverse variant="error">
+                        Button
+                    </Button>
+                    <Button isInverse variant="white">
+                        Button
+                    </Button>
+                </SnapshotWrapper>
+            </Row>
+
+            <H5>with an icon </H5>
+            <Row>
+                <SnapshotWrapper data-test="button_inverse_icon">
+                    <Button isInverse icon="PLUS" variant="success">
+                        Button
+                    </Button>
+                    <Button isInverse icon="PLUS" variant="info">
+                        Button
+                    </Button>
+                    <Button isInverse icon="PLUS" variant="warning">
+                        Button
+                    </Button>
+                    <Button isInverse icon="PLUS" variant="error">
+                        Button
+                    </Button>
+                    <Button isInverse icon="PLUS" variant="white">
+                        Button
+                    </Button>
+                </SnapshotWrapper>
+            </Row>
+
+            <H5>with loading </H5>
+            <Row>
+                <SnapshotWrapper data-test="button_inverse_loading">
+                    <Button isInverse isLoading variant="success">
```

Comment:
personally, I d suggest generating this long list from a series of config arrays with multiple nested map() functions. This would reduce code and increase maintainability dramatically.

Reply:
Looks good, the only problem is we have some props which can not be combined. It's because our props on Button are still a litle weird, bad concept ... for example isTransparent Button is for me another variant not a diferent prop (now you can add variant to button but when you use isTransparent, the variant is overwritten). 

There is also the isDisabled prop which for me should do only one thing - set the opacity to 0.5 ... so the style of the button will be the same only the opacity and cursor changes.

But i want to merge this big PR and do this in another. I generated it for now with 3 arrays with some conditions. When the props will make sense and could be combined then i could use this awesome package :)